wcbench.sh

```diff
@@ -610,6 +676,54 @@ install_opendaylight()
     add_to_featuresBoot "odl-openflowplugin-drop-test"
 }
 
+install_opendaylight_ubuntu()
```

Comment:
Same feedback as the `install_cbench_*` function. We're adding a ton of unnecessary code duplication buy splinting this into Ubuntu and Fedora functions, since (I think) the only part that's OS-specific is the `yum` vs `apt-get` install step. So, I suggest not creating a different function and instead adding a conditional to check which OS we're on and use the correct install command.